web/modules/custom/dbcdk_community_moderation/dbcdk_community_moderation.module

```diff
@@ -8,9 +8,17 @@
  * Implements hook_user_login().
  */
 function dbcdk_community_moderation_user_login(\Drupal\user\Entity\User $account) {
-  // Redirect moderators to the flagged content list.
-  if (in_array('admin_moderator', $account->getRoles())) {
+  // Redirect users with specific focus to their primary page.
+  // Moderators go to the flagged content page.
+  if ($account->getRoles(TRUE) == ['admin_moderator']) {
```

Comment:
If I'm correct, then this only assumes that a user has one role. If the user had moderator AND library roles, then they would just hit the user page. Shouldn't we use `in_array`?